tests/unit_tests/common/test_tuple.cpp

```diff
@@ -331,5 +332,12 @@ namespace gridtools {
             EXPECT_EQ(a, 42);
             EXPECT_EQ(b, 2.5);
         }
+
+#if not GT_NVCC_WORKAROUND_1766
```

Comment:
I think it's best to use the special character operators for consistency:
```suggestion
#if !GT_NVCC_WORKAROUND_1766
```